ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -676,6 +716,18 @@
 			name = Products;
 			sourceTree = "<group>";
 		};
+		3A3CB9071EF76B01008DAC6D /* AllAboutOlafWidget */ = {
+			isa = PBXGroup;
+			children = (
+				3A3CB9081EF76B01008DAC6D /* TodayViewController.h */,
+				3A3CB9091EF76B01008DAC6D /* TodayViewController.m */,
+				3A3CB90B1EF76B01008DAC6D /* MainInterface.storyboard */,
+				3A3CB90E1EF76B01008DAC6D /* Info.plist */,
+			);
+			name = AllAboutOlafWidget;
+			path = "AllAboutOlafWidget";
```

Comment:
If you do "stuff" with Xcode, it will remove this line. Please do stuff with xcode until it removes this.